### ISSUE TITLE
Stops when user:pass is asked

### ISSUE THREAD

**Issue by aacos24653** [temadol/gmail_generator#4]
To avoid user having to write user:pass for the proxies, would be good to change coding to IP:port:user:pass, this way people can use resi.

But as of right now it crashes when it asks for user:pass

**Comment by bernon** [comment]
yes, that's right! same issue and looking for something reliable with resi proxies :( probably this code works on older version of firefox "Since Firefox 90 'ftpProxy' is no longer supported" - when I'm using user:pass this error pops up

**Comment by D4RKH0R1Z0N** [comment]
What api key should I enter?
